### ISSUE TITLE
Having trouble using with grunt-jasmine-nodejs

### ISSUE THREAD

**Issue by jameswilddev** [just-boris/jasmine-collection-matchers#3]
I'm trying to use this with https://www.npmjs.com/package/grunt-jasmine-nodejs on Windows, and when I require jasmine-collection-matchers I get:

```
Warning: ENOENT, no such file or directory '{your workspace}\src' Use --force to continue.
```

I poked around in node_modules and found a src directory in the jasmine-collection-matchers folder.  When copied to the same folder as the gruntfile, everything works as expected, though it does mean I have bits of node modules floating inside my project.  This would be especially problematic for people who store their project source in a "src" folder.  Not sure if this is an issue with jasmine-collection-matchers or grunt-jasmine-nodejs.

Cheers, James.

**Comment by just-boris** [comment]
confirm bug.
It seems to be fixed via https://github.com/just-boris/jasmine-collection-matchers/commit/fff874ca6f89be28006d99aec6735d9ce40b5b67

**Comment by just-boris** [comment]
released version `0.1.0`. It should work fine.

**Comment by jameswilddev** [comment]
Thanks.  I'll give that a try soon.  Awesome little library!  Makes Jasmine a lot nicer with collections.
